zombienet_tests/async_backing/async-backing.feature

```diff
@@ -0,0 +1,31 @@
+Description: Async Backing Version Test
+Network: ./async-backing-test.toml
+Creds: config
+
+# General
+alice: is up
+bob: is up
+
+# Check authority status 
+alice: reports node_roles is 4
+bob: reports node_roles is 4
+
+# Check peers
+alice: reports peers count is at least 2 within 20 seconds
+bob: reports peers count is at least 2 within 20 seconds
+
+# Parachain registration
+alice: parachain 100 is registered within 225 seconds
+bob: parachain 100 is registered within 225 seconds
+
+# Ensure parachain progress
+alice: parachain 100 block height is at least 10 within 200 seconds
+bob: parachain 100 block height is at least 10 within 200 seconds
+
+# Finality lag
+alice: reports polkadot_parachain_approval_checking_finality_lag is 0
```

Comment:
Instant approval will happen since the validator count is too low, so likely it is better to scale this test to something like 5 validators.

Reply:
In that case, it makes more sense to keep the test simple and avoid testing finality lag here as the sole purpose of the test is to check that blocks are being produced with two different polkadot versions, and not the performance wrt. which the blocks are finalised.